cli/src/Api.ts

```diff
@@ -495,41 +495,41 @@ export default class Api {
     return useCache && this._cdClassesCache
       ? this._cdClassesCache
-      : (this._cdClassesCache = await this.entriesByIds<ClassId, Class>(this._api.query.contentDirectory.classById))
+      : (this._cdClassesCache = await this.entriesByIds<ChannelId, Channel>(this._api.query.content.channelById))
   }
 
   availableCuratorGroups(): Promise<[CuratorGroupId, CuratorGroup][]> {
-    return this.entriesByIds<CuratorGroupId, CuratorGroup>(this._api.query.contentDirectory.curatorGroupById)
+    return this.entriesByIds<CuratorGroupId, CuratorGroup>(this._api.query.content.curatorGroupById)
   }
 
   async curatorGroupById(id: number): Promise<CuratorGroup | null> {
-    const exists = !!(await this._api.query.contentDirectory.curatorGroupById.size(id)).toNumber()
-    return exists ? await this._api.query.contentDirectory.curatorGroupById<CuratorGroup>(id) : null
+    const exists = !!(await this._api.query.content.curatorGroupById.size(id)).toNumber()
+    return exists ? await this._api.query.content.curatorGroupById<CuratorGroup>(id) : null
   }
 
   async nextCuratorGroupId(): Promise<number> {
-    return (await this._api.query.contentDirectory.nextCuratorGroupId<CuratorGroupId>()).toNumber()
+    return (await this._api.query.content.nextCuratorGroupId<CuratorGroupId>()).toNumber()
   }
 
-  async classById(id: number): Promise<Class | null> {
-    const c = await this._api.query.contentDirectory.classById<Class>(id)
+  async channelById(channelId: number): Promise<Channel | null> {
+    const c = await this._api.query.content.channelById<Channel>(channelId)
     return c.isEmpty ? null : c
   }
 
-  async entitiesByClassId(classId: number): Promise<[EntityId, Entity][]> {
-    const entityEntries = await this.entriesByIds<EntityId, Entity>(this._api.query.contentDirectory.entityById)
-    return entityEntries.filter(([, entity]) => entity.class_id.toNumber() === classId)
+  async videosByChannelId(channelId: number): Promise<[VideoId, Video][]> {
+    const videoEntries = await this.entriesByIds<VideoId, Video>(this._api.query.content.videoById)
+    return videoEntries.filter(([, video]) => video.in_channel.toNumber() === channelId)
   }
 
-  async entityById(id: number): Promise<Entity | null> {
-    const exists = !!(await this._api.query.contentDirectory.entityById.size(id)).toNumber()
-    return exists ? await this._api.query.contentDirectory.entityById<Entity>(id) : null
+  async videoById(videoId: number): Promise<Video | null> {
+    const exists = !!(await this._api.query.content.entityById.size(videoId)).toNumber()
+    return exists ? await this._api.query.content.videoById<Video>(videoId) : null
   }
 
-  async dataByContentId(contentId: ContentId): Promise<DataObject | null> {
+  async dataByContentId(contentId: number): Promise<DataObject | null> {
     const dataObject = await this._api.query.dataDirectory.dataByContentId<Option<DataObject>>(contentId)
```

Comment:
The query now returns `DataObject` (instead of `Option<DataObject>`) - to check whether it actually exists you'd need to use either `!dataObject.isEmpty` or `!!(await this._api.query.dataDirectory.dataByContentId(contentId).size())`

Reply:
Done, thank you for pointing that issue https://github.com/Joystream/joystream/commit/91fa8e99eb0d25e4f1f2b2bbb6b504860634bfc3#diff-a7a049ad43ba0426653943dc5211d8aea8c49e82a5ecef553d1aa18e7cf48008R529